File: high-school/fourth-year/first-semester/tpsit/Gioco/src/gioco/Random.java
package gioco;

/**
 * **************************************************************
 * RANDOM
 *
 * @author Gazi Adnan Latif
 * @brief oggetto random: genera casualmente una faccia del dado.
 * @date 11/04/2017
 * ****************************************************************
 */
public class Random {

    private int inf;
    private int sup;

    public Random(int inf, int sup) {
        this.inf = inf;
        this.sup = sup;
    }

    public int next() {
        return (int) Math.floor(Math.random() * (sup - inf) + 1);
    }
}

File: high-school/fift-year/first-semester/informatica/ProvaSingolaGazi/public_html/Random.js
function Random(n) {
	for (var i = 0; i < 90; i++) {
		var rand = Math.floor(Math.random() * 90);
		var appoggio = n[i];
		n[i] = n[rand];
		n[rand] = appoggio;
	}
	return n;
}

